src/test/shell/integration/startup_options_test.sh

```diff
@@ -82,4 +82,23 @@ function test_autodetect_server_javabase() {
   bazel --noautodetect_server_javabase version &> $TEST_log || fail "Should pass"
 }
 
+# Below are the regression tests for Issue #7489
+function test_multiple_bazelrc_later_overwrites_earlier() {
+  tmpdir1=$(cd $(mktemp -d) &> /dev/null && pwd -P)
+  tmpdir2=$(cd $(mktemp -d) &> /dev/null && pwd -P)
+  echo "common --repository_cache=$tmpdir1" > 1.rc
```

Comment:
Can you use a different flag? Running into problems with import where our internal version doesn't recognize `--respository_cache`. Same for other uses of `--repository_cache`. LMK if you want some suggestions (I'd have to look around the code some).

Reply:
`--repository_cache` was one of the few, if not the only `bazel info` key that will be read from bazelrc file, so had to be creative to test with `--color=yes/no` instead.